internal/webui/public/index.html

```diff
@@ -46,9 +46,9 @@
 
     <div class="container mt-5">
       <h1>Welcome to the Installer!</h1>
-      <p>Enter your <a href="https://kairos.io/docs/reference/configuration/" target="_blank">cloud-config config configuration file</a> and select the device to install on:</p>
+      <p>Enter your <a href="/local/docs/reference/configuration/" target="_blank">cloud-config config configuration file</a> and select the device to install on:</p>
       <div class="alert alert-info" role="alert">
-        Need help with your cloud config? Check out our <a href="https://kairos.io/docs/examples/" class="alert-link">documentation on where to find examples</a>.
+        Need help with your cloud config? Check out our <a href="/local/docs/examples/" class="alert-link">documentation on where to find examples</a>.
```

Comment:
we do have another link to the docs in the footer, maybe worth to redirect also to the local copy in that case?